### ISSUE TITLE
loss

### ISSUE THREAD

**Issue by xzltc** [lavendelion/YOLOv1-from-scratch#2]
首先感谢谢谢博主的代码，有点小疑惑
loss那块的代码中，计算bbox中不包含物体误差的confidence误差那一项C-iou是不是应该改为C-0。
我看原文里写的  _If no pred object exists in that cell, the confidence scores should be zero._ 
```python
if iou1 >= iou2:
    coor_loss = coor_loss + 5 * (torch.sum((self.pred[i, 0:2, m, n] - labels[i, 0:2, m, n]) ** 2) \
                + torch.sum((self.pred[i, 2:4, m, n].sqrt() - labels[i, 2:4, m, n].sqrt()) ** 2))
    obj_confi_loss = obj_confi_loss + (self.pred[i, 4, m, n] - iou1) ** 2
    noobj_confi_loss = noobj_confi_loss + 0.5 * ((self.pred[i, 9, m, n] - 0) ** 2)
else:
    coor_loss = coor_loss + 5 * (torch.sum((self.pred[i, 5:7, m, n] - labels[i, 5:7, m, n]) ** 2) \
                + torch.sum((self.pred[i, 7:9, m, n].sqrt() - labels[i, 7:9, m, n].sqrt()) ** 2))
    obj_confi_loss = obj_confi_loss + (self.pred[i, 9, m, n] - iou2) ** 2
    noobj_confi_loss = noobj_confi_loss + 0.5 * ((self.pred[i, 4, m, n] - 0) ** 2)
class_loss = class_loss + torch.sum((self.pred[i, 10:, m, n] - labels[i, 10:, m, n]) ** 2)
```
